src/main/resources/application.properties

```diff
@@ -2,4 +2,9 @@ spring.output.ansi.enabled=ALWAYS
 x2.user=apiuser
 x2.password=password
 x2.baseUrl=http://127.0.0.1/x2engine/index.php/api2/
-x2.contactsUrl=Contacts
\ No newline at end of file
+x2.contactsUrl=Contacts
```

Comment:
Yes it's for CRM. This will be removed along with the CRM repository and tests. 
I left this so that all tests did not break down and we could gently move to a new database.